### DISCUSSION TITLE
click and open tracking

### DISCUSSION THREAD

**Discussion by reetikpant** [postalserver/postal#1885] [Help with using Postal]
Can anyone help me how to enable this feature? I saw documentation, didn't understand much. I want to know how I can enable click and open feature, i dont get where to put X-postal-track and also that click.domain.com, how it works, if anyone can help please come forward

**Comment by willpower232** [comment]
Click tracking needs to happen on a domain that is similar to the one you send from so that the email does not look bad by linking to unrelated things.

You need to set up these domains in your nginx or caddy (pointing at the same port 5000 but with `X-Postal-Track-Host: 1`) and then you need to create them in the Postal UI.

**Reply by reetikpant** [reply]
how to add X-Postal-Track-Host: 1 in caddy??

also X-Postal-Track-Host: 1 - its same as it is right? i dont have to change anything??
